tokio/src/runtime/basic_scheduler.rs

```diff
@@ -76,6 +80,10 @@ pub(crate) struct Config {
 
     /// Callback for a worker unparking itself
     pub(crate) after_unpark: Option<Callback>,
+
+    #[cfg(tokio_unstable)]
+    /// How to respond to unhandled task panics.
+    pub(crate) unhandled_panic: crate::runtime::UnhandledPanic,
```

Comment:
nit/TIOLI: perhaps this should be called `on_unhandled_panic` or something, since there's also a field called `unhandled_panic` that indicates _whether_ there was an unhandled panic, and that seems confusable?